app/javascript/packages/password-strength/password-strength-element.ts

```diff
@@ -160,11 +160,14 @@ class PasswordStrengthElement extends HTMLElement {
       const result = zxcvbn(this.input.value, this.forbiddenPasswords);
       const score = this.#getNormalizedScore(result);
       this.setAttribute('score', String(score));
+      this.input.setAttribute('aria-describedby', 'password-strength password-description');
```

Comment:
The reference to `password-description` here likely violates one of the guiding principles to ensuring that these components are reusable and make no assumptions about the page they're used in.

Via [`packages/README.md`](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/README.md):

>As much as possible, their behaviors should be reusable and **make no assumptions about particular application pages** or configuration.

I think we could probably do some sort of toggling of `password-strength` into and out of the attribute value instead?

e.g.

```suggestion
      this.input.setAttribute(
        'aria-describedby',
        [this.input.getAttribute('aria-describedby'), 'password-strength].join(' '),
      );
```

And below:

```ts
this.input.setAttribute(
  'aria-describedby',
  this.input.getAttribute('aria-describedby').replace(/\s*password-strength\s*/, ''),
);
```

Reply:
I see how that might cause some problems down the road.  Revised with [af6248b](https://github.com/18F/identity-idp/pull/11907/commits/af6248be83fa6cda7f34f8c55ca8a33b0dde7a86)